dimagi/pages/templates/sections/pillar/mobile_data_collection/chapter4.html

```diff
@@ -4,9 +4,9 @@
 <article class="epic blue-theme" id="chapter-4">
 
   <section class="epic-cover"
-           data-lazybg="true">
+           data-lazybg="true">    
```

Comment:
in future PRs could you please make sure that these white space changes are not in the diffs?

Reply:
Sure, I will make assure in future PRs this trailing spaces are not there in any of the diffs.